### PR TITLE
Temp deletion

### PR DESCRIPTION
**Pull request by sumedhtadimeti**

### CODE REVIEW

**Comment by Buzzec** [comment]
Is this ready for review? If not make it a draft.

**Comment by sumedhtadimeti** [comment]
> Is this ready for review? If not make it a draft.
it's ready for review

File: app/models/user.rb [hunk 1]
```diff
@@ -18,6 +18,8 @@ def user_type_pretty
       "Admin"
     end
   end
+  
+  # hides the user instead of destroying the user
```
Comment (Buzzec) [inline app/models/user.rb:22]: Dangling comment

File: config/routes.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
   resources :committees
   resources :attendance_committee
-  resources :users
+  # resources :users
```
Comment (Buzzec) [inline config/routes.rb:8]: Remove this

**Review by Buzzec — CHANGES_REQUESTED** [review]

**Review by Buzzec — APPROVED** [review]

**Review by Luke2v52 — CHANGES_REQUESTED** [review]
Do not merge with master

File: app/views/layouts/_nav_bar.html.erb [hunk 3]
```diff
@@ -26,9 +28,8 @@
               Members
             </a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <%= link_to "Current Members", users_path, class: "dropdown-item" %>
-              <%= link_to "Pending Members", users_pending_path, class: "dropdown-item" %>
-            </div>
```
Comment (Luke2v52) [inline app/views/layouts/_nav_bar.html.erb:31]: You removed the closing div

**Review by Luke2v52 — CHANGES_REQUESTED** [review]

**Review by Luke2v52 — APPROVED** [review]
I fixed the issues

**Pull request merged**